stdlib/LinearAlgebra/src/bidiag.jl

```diff
@@ -724,3 +724,5 @@ function eigvecs(M::Bidiagonal{T}) where T
     Q #Actually Triangular
 end
 eigen(M::Bidiagonal) = Eigen(eigvals(M), eigvecs(M))
+
+Base._sum(A::Bidiagonal, ::Colon) = sum(A.dv) + sum(A.ev)
```

Comment:
Why overload `_sum` instead of just `sum`?

Reply:
Because of the way `sum` is defined in terms of `_sum` here: https://github.com/JuliaLang/julia/blob/5fa469bf21c2c015865e861b21432a6ef93a2569/base/reducedim.jl#L653. Do you think this is not the best way of doing it?